backend/db-init/SQL_Drop.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+// Initializing DB login variables -- compliant with the login information decided on by the team
+$database_host = 'localhost';
```

Comment:
`localhost` tried to connect to a non-existent socket on my Mac. Changing this to `127.0.0.1` worked.